tests/test_quirks.py

```diff
@@ -837,6 +837,77 @@ def check_for_duplicate_cluster_ids(clusters) -> None:
         check_for_duplicate_cluster_ids(ep_data.get(OUTPUT_CLUSTERS, []))
 
 
+@pytest.mark.parametrize("quirk", ALL_QUIRK_CLASSES)
+def test_suspicious_cluster_moves(quirk: CustomDevice) -> None:
+    """Verify that no quirks do suspicious moves or copy/pastes of clusters."""
+    for ep_id, ep_data in quirk.replacement[ENDPOINTS].items():
+        # Originals
+        orig_in_clusters = set(
+            quirk.signature.get(ENDPOINTS, {}).get(ep_id, {}).get(INPUT_CLUSTERS, [])
+        )
+        orig_out_clusters = set(
+            quirk.signature.get(ENDPOINTS, {}).get(ep_id, {}).get(OUTPUT_CLUSTERS, [])
+        )
+
+        # New
+        new_in_clusters = {
+            cluster.cluster_id if isinstance(cluster, zcl.Cluster) else cluster
```

Comment:
This will never return `cluster.cluster_id`, as `cluster` is never an instance. It's only a subclass.
This will cause classes to end up in the set, causing more failures than we should have.

Reply:
Can we try this:
```suggestion
            cluster if isinstance(cluster, int) else cluster.cluster_id
```

Reply:
Tested this locally now and can confirm this brought the test failures from 73 down to 40 for this entire test.

Reply:
Pushed this in [`04dddc2` (#4259)](https://github.com/zigpy/zha-device-handlers/pull/4259/commits/04dddc22b51e3f3c9d25b324e8694385183b34bd) for an easier overview of the actual failures. Feel free to change the fix if you like some other approach better.